modules/sdk-core/src/bitgo/utils/opengpgUtils.ts

```diff
@@ -35,6 +40,26 @@ export async function getBitgoGpgPubKey(bitgo: BitGoBase): Promise<Key> {
   return await readKey({ armoredKey: bitgoPublicKeyStr });
 }
 
+/**
+ * Verifies the primary user on a GPG key using a reference key representing the user to be checked.
+ * Allows a verification without a date check by wrapping verifyPrimaryUser of openpgp.
+ * @param {Key} pubKey gpg key to check the primary user of.
+ * @param {Key} primaryUser gpg key of the user to check.
+ * @param {boolean} checkDates If false, disable date checks in the openpgp call to check the primary user.
+ * @return {KeyValidityDict} list of users checked and whether each passed as a primary user in pubKey or not.
+ */
+export async function verifyPrimaryUserWrapper(
```

Comment:
Nit, the naming can be improved. `Wrapper` does not actually tell callers what this is doing or why its being wrapped. 

`verifyPrimaryUserWithDateCheck` might be better.